pkgs/top-level/all-packages.nix

```diff
@@ -10806,7 +10806,11 @@ with pkgs;
 
   rnnoise = callPackage ../development/libraries/rnnoise { };
 
-  rnnoise-plugin = callPackage ../development/libraries/rnnoise-plugin {};
+  rnnoise-plugin = callPackage ../development/libraries/rnnoise-plugin {
+    # Use `apple_sdk_11_0` because `apple_sdk.libs` does not provide `simd`
```

Comment:
```suggestion
  # Use `apple_sdk_11_0` because `apple_sdk.libs` does not provide `simd`
  rnnoise-plugin = darwin.apple_sdk_11_0.callPackage ../development/libraries/rnnoise-plugin {
```

This must be used